.github/workflows/terraform_provider.yml

```diff
@@ -0,0 +1,258 @@
+name: Terraform Provider Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - .github/workflows/terraform_provider.yml
+      - .golangci.yml
+      - aws/**
+      - awsproviderlint/**
+      - docs/index.md
+      - docs/data-sources/**
+      - docs/guides/**
+      - docs/resources/**
+      - go.sum
+      - GNUmakefile
+      - main.go
+      - staticcheck.conf
+      - website/**
+
+env:
+  AWS_DEFAULT_REGION: us-west-2
+  GO_VERSION: "1.14"
+  GO111MODULE: on
+  TERRAFORM_VERSION: "0.12.24"
```

Comment:
Should this be 0.12.25?

Reply:
Looks like this came out between submission and review -- I can update it here to ensure the PR passes CI still, but maybe the Terraform CLI version can be either:

- Read from a consistent file (similar to `.go-version`)
- Fetched in `hashicorp/setup-terraform` by it supporting the latest of a minor version (e.g. just passing `"0.12") (I'll submit an issue over there or play with this after merging)